### DISCUSSION TITLE
Monthly call scheduling and agenda

### DISCUSSION THREAD

**Discussion by jdidion** [openwdl/wdl#397] [General]
# Scheduling

* I suggest we start with a 90 minute call once per month
* Members span many time zones (Western Europe west to Austraila) - there likely won't be a time that's ideal for everyone
    * 6 am AEST == 4pm EST == 11 pm CEST
    * 1 pm AEST == 11 pm EST == 6 am CEST
    * 9 pm AEST == 7 am EST == 2 pm CEST
* We could do the call on rotation between two or three times
* The call should be recorded so anyone who can't make it can catch up
* The call should only be for discussion - voting should still happen on GitHub

# Draft agenda for first meeting

* WDL 2.0 status and ratification schedule
* Discussion of open issues
    * Type coercion (#396, #392, #391, #373)
    * WDL publication
* Presentation of new proposals (if any)
* Demos
    * wdlTools

**Comment by mlin** [comment]
:+1: the suggested timeslots generally LGTM; agree with the idea of rotating timeslots to spread the hurt

Re "Governance call" -- this would be open to all, right?

**Comment by jdidion** [comment]
I figured it would only be of interest to the governance folks but I don't see any reason not to let anyone join.

**Comment by illusional** [comment]
I'm not part of the governance team but interested in joining, so I'd appreciate it being open for anyone to join. The term "Governance call" does seem less friendly to someone who was considering coming to the call.

**Comment by patmagee** [comment]
The time slots look good to me, i can easily make 2/3 of them in general. I would also propose we change thebname from governance call to something like:

- cadence call
- progress call
- monthly call

**Comment by rhpvorderman** [comment]
The time slots do not look very friendly to Europeans. 6 AM and 11PM are times that are hard to meet. I do appreciate that it is hard to come up with time slots were at least 2 out of 3 are possible. Day light savings is a pain as well.

Anyway, since the calls are recorded and the decisions are made on Github anyway, it is not that big of a deal not being able to join all slots. And having a monthly call that can be only attended one in three times is much better than having nothing at all.

**Comment by DavyCats** [comment]
~~Most ideal time according to [this meeting planner tool](https://www.timeanddate.com/worldclock/meetingtime.html?day=27&month=8&year=2020&p1=43&p2=16&p3=240) would be at 11:00 UTC (7:00 in Boston, 13:00 in Amsterdam, 21:00 in Sydney). It's the only time none of the timezones are in the "sleeping hours" range.~~ But yeah, the fact that there are people spread out across the entire world makes it hard to pick a time that would work for everyone.

EDIT: I just recalled @mlin is on Hawaii, so actually [it's impossible to have no one in "sleeping hours"](https://www.timeanddate.com/worldclock/meetingtime.html?iso=20200827&p1=43&p2=16&p3=240&p4=103), that time I suggested above would be at 1:00 in Honolulu. Most ideal time then would probably be 21:00 UTC (17:00 Boston, 23:00 Amsterdam 😞, 7:00 Sydney, 11:00 Honolulu)

Also this only goes if the wider community would join the meeting, since I don't believe there is anyone from Europe in the core team.

**Comment by patmagee** [comment]
@DavyCats I think the idea is to have people from the community join the call either to observe or even contribute to the meetings. Worst case scenario, we coipy the GA4GH approach and have two rolling meeting slots, one friendly for our friends in the east, the other for our friends in the west.

**Comment by patmagee** [comment]
To move ahead with this, I propose we aim for the first governance call to be at the end of September, 2020. Potentially during the week of September 21st - 25th

**Comment by patmagee** [comment]
@jdidion i thought this would be a good topic to try out in discussions. discussions seem to be actually very similar to stack exchange in principal so this could be really useful for pur needs!

**Comment by mlin** [comment]
@DavyCats 1am is fine for me actually. I'm usually working on something WDL-related at that time :)
